contracts/src/spaces/facets/roles/Roles.sol

```diff
@@ -90,7 +84,41 @@ contract Roles is IRoles, RolesBase, Entitled {
     CreateEntitlement memory entitlement
   ) external {
     _validatePermission(Permissions.ModifyRoles);
-    _checkRoleExists(roleId);
     _removeRoleFromEntitlement(roleId, entitlement);
   }
+
+  // custom channel permissions
+  function createCustomChannelPermissions(
```

Comment:
Again, overrides, not custom

Reply:
Also, set, not create

Reply:
so set would have to remove already created permissions and add the new ones? what if you only want to remove a single permission? this would require multiple writes for a single change